### DISCUSSION TITLE
ERROR! The requested handler 'reboot' was not found in either the main handlers list nor in the listening handlers list

### DISCUSSION THREAD

**Discussion by FraGThieF** [techno-tim/k3s-ansible#112] [Questions] [answered]
Hello together,
i am trying to install k3s on 6 raspberries and i get this error message:
 ```
TASK [raspberrypi : Activating cgroup support] ********************************************************************************************************************************************************************************************
Mittwoch 28 September 2022  22:02:07 +0200 (0:00:00.102)       0:00:29.039 **** 
ok: [frag-k3s-w2]
ok: [frag-k3s-w1]
ok: [frag-k3s-m3]
ok: [frag-k3s-w3]
ERROR! The requested handler 'reboot' was not found in either the main handlers list nor in the listening handlers list
```
Anyone an idea what i have to do here?

Thank you so much

**Comment by FraGThieF** [comment, marked as answer]
strange: 
play reset.yml 
reboot everything.
play site.yml
works.

**Comment by marpi82** [comment]
I have similar issue.

When I run site.yaml on my laptop to create k3s on proxmox lxc containers I'am getting:
<code>ERROR! The requested handler 'Reboot containers' was not found in either the main handlers list nor in the listening handlers list</code>
or/and
<code>ERROR! The requested handler 'Reboot server' was not found in either the main handlers list nor in the listening handlers</code>

**Comment by marpi82** [comment]
Yes, but I have to run the playbook six or more times before it reaches the
end. Each time it stops with this error, but keeps getting further and
further away.  Same during reset - but probably only once or twice.

The problem is that I wanted to use this playbook to learn kubernetes, I
admit, quite often I break something and then all this repetition becomes
very tedious.

So I would like to know what causes this behavior of the script and correct
it.


niedz., 13 sie 2023, 19:35 użytkownik Derek Jacobs ***@***.***>
napisał:

**Reply by timothystewart6** [reply]
I would recommend just trying with VMs.  LXC should work too but it's newer and less tested.

**Reply by marpi82** [reply]
I have found a solution and make a PR #348

**Reply by timothystewart6** [reply]
Thank you so much!
